linera-storage/src/lib.rs

```diff
@@ -288,6 +295,24 @@ where
         Ok(value.with_hash_unchecked(hash))
     }
 
+    async fn read_values(
```

Comment:
I wonder whether we should give this a different name, because e.g. `write_values` takes a list of values, whereas this reads _consecutive_ values. Maybe `read_value_chain`? `read_values_up_to`? I'll let others chime in.

Reply:
how about `read_values_downward(..., from : ..., ...)` ? (`to` looks a bit weird to me)

Reply:
Looks good to me. :+1: